gradle/verification-metadata.xml

```diff
@@ -82,13 +82,12 @@
          <trusted-key id="0a123c1ed3f13a6a0140e166c71fb765cd9de313" group="org.apache.ant"/>
          <trusted-key id="0cb5871fb7bf3b351614bbf6ca85ffe638d4407a" group="it.unimi.dsi" name="fastutil"/>
          <trusted-key id="0e18eae07b7774eac5db3f2113bb90ce8eafbe37" group="com.microsoft.playwright"/>
+         <trusted-key id="7b79add11f8a779fe90fd3d0893a028475557671" group="com.gradle" name="gradle-enterprise-gradle-plugin"/>
```

Comment:
🤔 I'd probably have replaced the section for the trusted key for `com.gradle` with the new key id, though I am also fine to do it like this. We only need to remember to give more trust to this key should we use more artifacts from the `com.gradle` group.

```
         <trusted-key id="0ef2cac6b3577539b7ab5bcc91aa2c186c4d7a86">
            <trusting group="com.gradle"/>
         </trusted-key>
```